packages/node-runtime-worker-thread/src/rpc.ts

```diff
@@ -48,30 +55,27 @@ function send(messageBus: RPCMessageBus, data: any): void {
   }
 }
 
-function serialize(data: unknown): string {
-  return `data:;base64,${v8.serialize(data).toString('base64')}`;
-}
-
-function deserialize<T = unknown>(str: string): T | string {
-  if (/^data:;base64,.+/.test(str)) {
-    return v8.deserialize(
-      Buffer.from(str.replace('data:;base64,', ''), 'base64')
-    );
-  }
-  return str;
-}
-
 function getRPCOptions(messageBus: RPCMessageBus): PostmsgRpcOptions {
   return {
     addListener: messageBus.on.bind(messageBus),
     removeListener: messageBus.off.bind(messageBus),
     postMessage(data) {
       if (isClientMessageData(data) && Array.isArray(data.args)) {
+        // We don't guard against serialization errors on the client, if they
```

Comment:
What does this mean?

Reply:
As this is a general RPC helper, I was writing comments using the server-client terminology: so like the client is whoever calls a method, server is whoever executes it and returns the result. What would you suggest instead? Local-remote I think is sometimes used for RPC, sounds better? 🤔

Reply:
Don't know, i think client server, if is used consistently is probably ok. I would understand better if we call worker, or child process the part that executes the code probably, as is more in context, but by all means if we have client/server everywhere is fine.

What i missed here though is which part of the client is 'client' here? From the comment i thought that the serialization error is something that the browser-shell would have had to handle in a special way, but it seems the 'client part' of the rpc already does.

Reply:
So what I was trying to communicate  is that whoever is issuing an RPC request, will be responsible for handing the errors, whatever they are. If exposed method throws (on the server), we propagate the error back and re-throw, if caller throws (on the client, e.g., during serialization of arguments), we don't do anything and just throw. Re-reading this particular comment I think it's more confusing than helping, so I'll remove it

Reply:
Updated in 8a77045. Hopefully it's better now